### ISSUE TITLE
[Bug] Axes no longer effective on wood

### ISSUE THREAD

**Issue by Hunter-495** [quiqueck/BCLib#158]
### What happened?

Using this mod without Better end/Better nether or any other child mods will result in Axes no longer being effective on wood blocks... for some reason.

Steps to reproduce: Launch minecraft with BClib version 3.0.14 without any child mods, once in a world attempt to mine wood with an axe.

Expected result: Axes are effective tools for mining wood blocks.

### BCLib

3.0.14

### Minecraft

1.20.1

**Comment by DawsonBodenhamer** [comment]
YES after hours of troubleshooting, I have found out the bclib-3.0.14 mod was the one causing all my axes to be completely ineffective (and yes I had removed better nether and better end because they don't work with distant horizons, and that's when the issue started because I forgot to remove bclib

**Comment by HeavyLogic** [comment]
Same. bclib-3.0.14, fabric-api-0.92.1+1.20.1

**Comment by pawrott** [comment]
thank you dawson

**Comment by VertexBufferObject** [comment]
I have this same problem.

**Comment by 1469031954** [comment]
I have this same problem. Axe is not ineffective anymore.
